boundaries/index.json

```diff
@@ -214,6 +214,19 @@
             "lang:en_CA": "label"
         }
     },
+    {
+        "directory": "ottawa-wards",
+        "associations": [
+            {
+                "comment": "Member of Ottawa City Council",
+                "position_item_id": "Q47487122"
+            }
+        ],
+        "area_type_wikidata_item_id": "Q47486824",
+        "name_columns": {
+            "lang:en_CA": "WARD_NAME_"
```

Comment:
It looks like there is a French version under `WARD_NAME_1`, so it would be good to add that under `lang:fr_CA`